fix: postcss-normalize-unicode causes bugs in IE11

vitaliymaz opened the pull request:
Sorry, closed the previous PR by mistake. 

fixes #524

evilebottnawi commented on packages/postcss-normalize-unicode/src/index.js:
```diff
@@ -40,15 +43,34 @@ function unicode (range) {
+ */
+
+function hasLowerCaseUPrefixBug (browser) {
+    return ~['ie 8', 'ie 9', 'ie 10', 'ie 11'].indexOf(browser);
```
What about edge?

evilebottnawi replied:
@vitaliymaz can you great easy gist i test this using browserstack tomorrow?

vitaliymaz replied:
Yes, I will.

vitaliymaz replied:
I'm not able to work on this today. I will continue everything tomorrow.

vitaliymaz commented on packages/postcss-normalize-unicode/src/__tests__/index.js:
```diff
@@ -12,59 +12,68 @@ test(
     fixture('u+2b00-2bff'), // Miscellaneous Symbols and Arrows
-    fixture('u+2b??')
+    fixture('u+2b??'),
+    {env: 'chrome58'}
```
ie11 is in the list of browsersList by "defaults" selector.  http://browserl.ist/
So that means '+u' will be upper cased and standart test will fail.

evilebottnawi replied:
@vitaliymaz hm, okey, let's add test where ie11 in not browserslist

vitaliymaz replied:
Do you mean use something like {env: 'not ie'} instead of {env: 'chrome58'}?

evilebottnawi replied:
@vitaliymaz any

vitaliymaz commented:
@evilebottnawi, I've added the logic not to lowercase 'U' prefix for edge browsers before 16 version. Edge 16 works ok. Also I've added a new browser env to package.json - "not ie" for testing purposes and I'm not sure it is what you adviced.

vitaliymaz commented:
Yes, let's merge it.
